src/app/view/components/not-licensed/not-licensed.component.ts

```diff
@@ -1,10 +1,52 @@
-import { Component, ViewEncapsulation } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import {
+    UnlicensedPortletData,
+    DotLicenseService
+} from '@services/dot-license/dot-license.service';
+import { DotMessageService } from '@services/dot-messages-service';
+import { take, takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
 
 @Component({
-    encapsulation: ViewEncapsulation.None,
-
     selector: 'dot-not-licensed-component',
     styleUrls: ['./not-licensed.component.scss'],
     templateUrl: 'not-licensed.component.html'
 })
-export class NotLicensedComponent {}
+export class NotLicensedComponent implements OnInit, OnDestroy {
+    messagesKey: { [key: string]: string } = {};
+    unlicenseData: UnlicensedPortletData;
+
+    private destroy$: Subject<boolean> = new Subject<boolean>();
+
+    constructor(
+        private dotMessageService: DotMessageService,
+        private dotLicense: DotLicenseService
+    ) {}
+
+    ngOnInit() {
+        this.dotLicense.unlicenseData
```

Comment:
use merge or concat here.

Reply:
I won't fix this, because It'll change with the new messages service.

it will be done when this gets done, https://github.com/dotCMS/core/issues/18530